<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataDicWindow</class>
 <widget class="QWidget" name="DataDicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据字典管理模块</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    background-color: #f5f7fa;
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, sans-serif;
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QWidget" name="buttonContainer" native="true">
     <property name="styleSheet">
      <string notr="true">
       background-color: white;
       border-radius: 6px;
       padding: 8px;
       box-shadow: 0 1px 4px rgba(0,0,0,0.05);
      </string>
     </property>
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnBack">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #95a5a6;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #7f8c8d;
          }
         </string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #3a6ea5;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #4a90e2;
          }
         </string>
        </property>
        <property name="text">
         <string>新增字典</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #3a6ea5;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #4a90e2;
          }
         </string>
        </property>
        <property name="text">
         <string>编辑</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #e74c3c;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #c0392b;
          }
         </string>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSearch">
        <property name="minimumSize">
         <size>
          <width>276</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QLineEdit {
              padding: 8px 12px;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              font-size: 13px;
              background-color: white;
          }
         </string>
        </property>
        <property name="placeholderText">
         <string>输入类型、键、值等关键词...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #27ae60;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
         </string>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableDicts">
     <property name="styleSheet">
      <string notr="true">
       QTableWidget {
           font-size: 13px;
           background-color: white;
           alternate-background-color: #f8f9fa;
           border: 1px solid #e0e0e0;
           border-radius: 6px;
       }
       QHeaderView::section {
           background-color: #3a6ea5;
           color: white;
           padding: 8px;
           border: none;
       }
      </string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>字典类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>键</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>排序</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="paginationWidget" native="true">
     <property name="styleSheet">
      <string notr="true">
       background-color: white;
       border-radius: 6px;
       padding: 8px;
      </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="labelPageInfo">
        <property name="text">
         <string>第 1 页 / 共 1 页 (10 条)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnFirst">
        <property name="text">
         <string>首页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrev">
        <property name="text">
         <string>上一页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboPage"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string>下一页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLast">
        <property name="text">
         <string>末页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboPageSize">
        <item>
         <property name="text">
          <string>10 条/页</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20 条/页</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50 条/页</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
